taichi/transforms/compile_to_offloads.cpp

```diff
@@ -102,9 +102,12 @@ void compile_to_offloads(IRNode *ir,
   print("Offloaded");
   irpass::analysis::verify(ir);
 
-  irpass::cfg_optimization(ir, false);
-  print("Optimized by CFG");
-  irpass::analysis::verify(ir);
+  // comment out `cfg_optimization` temporarily
+  // to avoid optimizing some important operations
+  // when testing bit-level GlobalLoad/Store
+//  irpass::cfg_optimization(ir, false);
+//  print("Optimized by CFG");
+//  irpass::analysis::verify(ir);
```

Comment:
I believe this should be reset when the PR is ready for its final-round review.

Reply:
Yes, you are right, I did this to avoid some storing or loading operations being optimized when testing. BTW, do you know how to skip some IR optimization pass other than simply commenting them out?